src/main/java/com/exasol/releaserobot/main/ReleaseRobot.java

```diff
@@ -28,10 +29,12 @@ public ReleaseRobot(final ReleaseUseCase releaseUseCase, final ValidateUseCase v
 
     /**
      * Main entry point for all Release Robot's calls.
+     *
+     * @throws GitHubException
      */
     // [impl->dsn~rr-starts-release-only-if-all-validation-succeed~1]
     // [impl->dsn~rr-runs-release-goal~1]
-    public void run(final UserInput userInput) {
+    public void run(final UserInput userInput) throws GitHubException {
```

Comment:
this exception comes from the validate/release execution, and should be handled there, and be reflected in the reports. I would suggest leave it as it is and create a ticket for it.

Reply:
Created a ticket: https://github.com/exasol/release-robot/issues/76